internal/filter/filter_v2_test.go

```diff
@@ -0,0 +1,281 @@
+package filter_test
```

Comment:
I think you could use backticks to avoid backslashes across multiple strings in this file.